cmd/influxd/main.go

```diff
@@ -5,32 +5,54 @@ import (
 	"fmt"
 	_ "net/http/pprof"
 	"os"
+	"sync"
 	"time"
 
 	"github.com/influxdata/influxdb/cmd/influxd/launcher"
 	"github.com/influxdata/influxdb/kit/signals"
 	_ "github.com/influxdata/influxdb/query/builtin"
+	"github.com/influxdata/influxdb/telemetry"
 	_ "github.com/influxdata/influxdb/tsdb/tsi1"
 	_ "github.com/influxdata/influxdb/tsdb/tsm1"
 )
 
+var (
+	version = "dev"
```

Comment:
These are replaced during builds correct?

Reply:
Yup, these are to be replaced by goreleaser.